Advanced Tag Filtering for Metrics Summary (#16062)
DataDog/documentation: content/en/metrics/summary.md

The [Metrics Summary page][1] displays a list of your metrics reported to Datadog under a specified time frame: the past hour, day, or week. 

Search your metrics by metric name or tag using the **Metric** or **Tag** search fields:

{{< img src="metrics/summary/tag_advancedfiltering.mp4" alt="Filter by Tag" video=true style="width:75%;">}}

> The tag filtering supports boolean and wildcard syntax such that you can quickly identify: 
> * Metrics are tagged with a particular tag key, team: `team:*`
> * Metrics that are missing a particular tag key, team: `NOT team:*`

Reviewer: ```suggestion
Tag filtering supports boolean and wildcard syntax so that you can quickly identify: 
* Metrics that are tagged with a particular tag key, for example, `team`: `team:*`
* Metrics that are missing a particular tag key, for example, `team`: `NOT team:*`
```

Author: Hey Bryce, I'm seeing the new text rendering below the mp4 element so I'm unable to replicate what you're seeing unfortunately
